test/tc_base_profiles_test.go

```diff
@@ -78,6 +78,15 @@ spec:
 	e.kubectl("create", "-f", helloProfileFile.Name())
 	defer e.kubectl("delete", "-f", helloProfileFile.Name())
 
+	e.logf("Waiting for profile to be reconciled")
+	for i := 0; i < 20; i++ {
+		output := e.kubectl("get", "sp", "hello")
+		if e.Contains(output, "hello") {
+			break
+		}
+		time.Sleep(time.Second)
```

Comment:
would be nice to add conditions to the SeccompProfile object, that way we could just do `kubectl wait --for condition=ready`

Reply:
Hm, right now we save the profile to disk after the CRD got created. This means we have a time gap which we did not cover right now. We could add the status `ready` to CRDs, where all profiles have been reconciled successfully.

To do that we would need some kind of server instance which collects that information from each spod.

Reply:
@saschagrunert that's exactly the work that @jhrozek is doing in https://github.com/kubernetes-sigs/security-profiles-operator/pull/361 I guess then all we need is the conditions there.

Reply:
Yeah, sounds reasonable. :+1: